frontend-api/src/main/java/com/cgdpd/library/frontendapi/controller/ViewController.java

```diff
@@ -23,4 +26,10 @@ public class ViewController {
     public BookViewDto detailedBookDto(@PathVariable Isbn13 isbn13) {
         return bookMapper.mapToBookView(bookService.getDetailedBookDtoByIsbn13(isbn13));
     }
+
+    @GetMapping("/book")
+    public Flux<BookViewDto> searchBooks(PaginationCriteria paginationCriteria, SearchBookCriteria searchBookCriteria) {
```

Comment:
We should be returning PagedResponse here, how will frontend know how many pages there are in total?